docs/en_US/Compression/v2_pruning_algo.rst

```diff
@@ -55,6 +55,29 @@ User configuration for Level Pruner
 
 .. autoclass:: nni.algorithms.compression.v2.pytorch.pruning.LevelPruner
 
+Performance Test
+^^^^^^^^^^^^^^^^
+
+.. list-table::
+   :header-rows: 1
+   :widths: auto
+
+   * - Model
+     - Accucacy
+     - Parameters
+     - FLOPs
+     - Sparsity
+   * - VGG-16
+     - 93.86%
+     - 14.98M
+     - 313.46M
+     - 
+   * - Pruned VGG-16
+     - 93.74%
+     - 14.98M
+     - 313.46M
```

Comment:
why parameters and flops do not change in the pruned model?

Reply:
This is fine-grained algorithm, it has no speedup procedure.